apps/web/src/studio/components/workflows/WorkflowsDetailPage.tsx

```diff
@@ -1,23 +1,67 @@
-import { SearchInput } from '@novu/design-system';
-import { Button } from '@novu/novui';
+import { Button, IconButton, Title } from '@novu/novui';
 import { css } from '@novu/novui/css';
-import { IconAddBox } from '@novu/novui/icons';
-import { Flex } from '@novu/novui/jsx';
+import {
+  IconCable,
+  IconOutlineBolt,
+  IconOutlineEmail,
+  IconOutlineNotifications,
+  IconPlayArrow,
+  IconSettings,
+} from '@novu/novui/icons';
+import { Flex, HStack, VStack } from '@novu/novui/jsx';
 import { PageContainer } from '../../layout/PageContainer';
 import { PageMeta } from '../../layout/PageMeta';
+import { StepNode } from './StepNode';
+import { WorkflowFloatingMenu } from './WorkflowFloatingMenu';
 
 export const WorkflowsDetailPage = () => {
-  const title = 'Example';
+  const title = 'Workflow name';
+
+  const handleSettingsClick = () => {};
+  const handleTestClick = () => {};
 
   return (
-    <PageContainer className={css({ colorPalette: 'mode.cloud' })}>
+    <PageContainer className={css({ colorPalette: 'mode.local' })}>
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Super clean ⚡‏